docs/getting-started/edfi-exchange/technology/ed-fi-ods-cloud-deploy-aws-suite-3/readme.md

```diff
@@ -0,0 +1,22 @@
+---
+---
+# Ed-Fi ODS/API Cloud Deployment for AWS (Suite 3)
+
+:::info
+The Ed-Fi Alliance no longer supports nor recommends using the "Ed-Fi ODS/API Cloud Deployment for AWS (Suite 3)" offering and is provided here for historical reference.  At the time this offering was developed, direct platform commands were necessary to support deployments out to cloud providers, which this offering relies heavily on.  Based on community feedback, we've migrated our deployments to utilize Docker as a primary deployment strategy, as all major cloud providers, along with on-premise methods to do so.  We recommend referring to [Docker Deployment 2.x](https://edfi.atlassian.net/wiki/spaces/EDFITOOLS/pages/24119348/Docker+Deployment+2.x) on TechDocs and the other Docker offerings here in the Exchange as ways to stand up Ed-Fi on the major cloud providers.
+:::
+
+## Description
+
+The Ed-Fi ODS / API deployment for Amazon Web Services consists of a set of Cloud Formation Templates and Lambda functions that make it easy to deploying a new copy of the Ed-Fi ODS / API to an Amazon Web Services tenant. The goal of these tools is to reduce the amount of time and effort needed to get a new instance of an Ed-Fi ODS/API up and running. This solution enables you to deploy the ODS / API on either PostgreSQL or Microsoft SQL Server.
+
+## Download
+
+* [Code & Documentation](https://edfi.atlassian.net/wiki/spaces/EXCHANGE/pages/22491023/Ed-Fi+ODS+API+Cloud+Deployment+for+AWS+-+Deployment+Guide)
```

Comment:
Update - provide `[Code]` link to GitHub repo and add link to the Deployment Guide here.

Reply:
Not sure what you mean here exactly. I added [Performance Test Code](./deployment-guide/readme.md) but I'm not sure what code this is suppose to be referencing.